schema/v1.0/ast/metrics.go

```diff
@@ -0,0 +1,34 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ast // import "go.opentelemetry.io/otel/schema/v1.0/ast"
+
+import "go.opentelemetry.io/otel/schema/v1.0/types"
+
+// VersionOfMetrics corresponds to a section representing a list of changes that happened
+// to metrics schema in a particular version.
+type VersionOfMetrics struct {
+	Changes []MetricsChange
+}
+
+// MetricsChange corresponds to a section representing metrics change.
+type MetricsChange struct {
+	RenameMetrics map[types.MetricName]types.MetricName `yaml:"rename_metrics"`
+	RenameLabels  *LabelMapForMetrics                   `yaml:"rename_labels"`
```

Comment:
`s/label/attribute/`

Reply:
Renamed variable and type, but kept yaml tag since it is specified in OTEP, can't change anymore.

Reply:
Spec SIG is happy with the change: https://github.com/open-telemetry/oteps/pull/181
I will fix this PR too.